st2common/st2common/models/db/stormbase.py

```diff
@@ -78,6 +78,23 @@ class StormBaseDB(StormFoundationDB):
     }
 
 
+class StormContentDB(StormBaseDB):
+    """Abstraction for a user content model from a pack."""
+    name = me.StringField(required=True)
+    pack = pack = me.StringField(
+        required=False,
+        help_text='Name of the content pack.',
+        unique_with='name')
+    file_uri = me.StringField(
```

Comment:
Can we use `artifact_uri` instead of `file_uri` to be consistent with the sensor model?